docs/pages/blog/2020-q1-update.md

```diff
@@ -0,0 +1,110 @@
+
+  - [Dmitriy Kovalenko](https://github.com/dmtrKovalenko), the author of @material-ui/pickers.
+    <img src="https://avatars0.githubusercontent.com/u/16926049" alt="dmitriy" style="max-width: 160px; margin: unset; margin-bottom: 24px; border-radius: 2px;" />
+
+  - [Damien Tassone](https://github.com/dtassone/), an experienced developer in the finance industry.
+    <img src="https://avatars0.githubusercontent.com/u/936978" alt="damien" style="max-width: 160px; margin: unset; margin-bottom: 24px; border-radius: 2px;" />
+
+### Growth between Q4 2019 and Q1 2020
+
+- 📦 From 3.2M to 4.8M downloads per month on npm.
+- ⭐️ From 53.3k to 56.2k stars, leave us yours 🌟.
+- 👨‍👩‍👧‍👦 From 1,581 to 1,720 contributors on GitHub.
+- 💰 Grew monthly financial support by 47%.
+- 🏢 From 3 to 5 full-time equivalent developers, spread among multiple financially supported core team members.
+
+## Our roadmap intent for Q2 2020
+
+We'll do our best, no guarantee!
+
+- 📣 We will analyze and publish the results of the "Material-UI Developer Survey 2020". You haven't contributed to it yet, you can [fill it with this link](https://forms.gle/TYWRdvgyZs4AhZNv8), thanks!
+- 🎨 We will release Figma assets for Material-UI.
+
+  <img src="/static/blog/2020-q1-update/figma.png" alt="figma" style="max-width: 160px; margin-bottom: 24px;" />
+
+- 🇨🇳 We will translate the documentation to Chinese at 100%. We are collaborating with [Danica Shen](https://github.com/DDDDDanica), a native speaker, to complete the effort. So far, we have translated 73% of the documentation and peer-reviewed 39%. You can help us out on [Crowdin](https://translate.material-ui.com/).
+
+  <img src="/static/blog/2020-q1-update/chinese.png" alt="chinese" style="max-width: 134px; margin-bottom: 24px;" />
+
+- 🔥 We will initiate our work on the [next major: v5](https://github.com/mui-org/material-ui/issues/20012).
+You can expect the following:
+
+  - A feature freeze on v4.
+  - The introduction of deprecation messages in the next v4 minors. These messages will help developers upgrade to v5.
+  - A progressive bug fixes freeze on v4, to the exception of security issues and important bugs.
+  - At least 6 months of work on v5 to get to a stable release (probably more). You can follow our [milestone](https://github.com/mui-org/material-ui/milestone/35).
+- 🧑‍💻 We will likely look for hiring a new full-time member on the core team to move faster with v5.
+- ⌗ We have announced, back in [October 2019](/blog/september-2019-update/#our-roadmap-intent-for-october), our intention to work on an advanced data grid component. The task was bigger than anticipated but we are making progress with it. It might take us 12 months to do it right. Damien is now leading and fully dedicated to this effort. You can follow our early-stage progress on [finui.io](https://finui.io/#/grid) and later integration into the mono-repository on [#18872](https://github.com/mui-org/material-ui/pull/18872).
```

Comment:
```suggestion
- ⌗ We announced, back in [October 2019](/blog/september-2019-update/#our-roadmap-intent-for-october), our intention to work on an advanced data grid component. The task was bigger than anticipated, but we are making progress. It might take us 12 months to do it right. Damien is now leading and fully dedicated to this effort. You can follow our early-stage progress on [finui.io](https://finui.io/#/grid) and later, integration into the mono-repository on [#18872](https://github.com/mui-org/material-ui/pull/18872).
```